### ISSUE TITLE
Dependencies with 'debug' – is there a policy?

### ISSUE THREAD

**Issue by ikonst** [EricssonResearch/openwebrtc#137]
I've noticed glib builds with `--enable-debug=no` regardless of `-r` or `-d` flag.
Is there a reason for this?
Are we trying to minimize debuggable packages to the higher-level ones, or is it just an omission?

**Comment by ikonst** [comment]
In fact, very few of the build-*.sh scripts respect the do_release variable at all.

**Comment by ikonst** [comment]
No, wait, it's just build-openwebrtc-gst-plugins.sh. That's the only dependency where debug/release makes any difference.

**Comment by sdroege** [comment]
FWIW, in cerbero we always build with debug symbols currently. -O2 -g or -Os -g. But no special configure parameters to things like glib to enable special debugging code.

**Comment by superdump** [comment]
Personally I like to have debug symbols always, but then I'm a developer. Can cerbero do release builds as well that are stripped?

**Comment by sdroege** [comment]
I think currently it can't, but adding that as a post-build step shouldn't be too hard.

**Comment by superdump** [comment]
Filed an issue against cerbero about that here: https://bugzilla.gnome.org/show_bug.cgi?id=747157
